Shared/UI/Views/SpeechBubbleView.swift

```diff
@@ -34,7 +34,11 @@ class SpeechBubbleView: View {
             return UIColor(cgColor: cgColor)
         }
         set {
+            // CALayers are implicitly animated. Disable animations so the color changes immediately.
+            CATransaction.begin()
+            CATransaction.setDisableActions(true)
             self.bubbleLayer.fillColor = newValue?.cgColor
+            CATransaction.commit()
```

Comment:
As their name suggests, Core Animation Layers animate their values implicitly. Because of this, the message bubble colors weren't changing immediately which caused problems when cells were recycled.

This little incantation disables the animations when setting the fill color.